Add Windows installation to the documentation (#329)
physiopy/phys2bids: docs/contributing.rst

    If you want to check, type ``python --version`` in a terminal.

> Linux, mac  and Windows developer installation

Reviewer: ```suggestion
Linux, Mac  and Windows developer installation
```

------------------------------------

Be sure to have ``git`` and ``pip`` installed. Fork the phys2bids repository to yours,then open a terminal and run::

	git clone https://github.com/{username}/phys2bids.git

> 	# or in case you have ssh key:

Reviewer: ```suggestion
	# or in case you have an ssh key set up:
```

	git clone git@github.com:{username}/phys2bids.git

> 	# we also recommend to set up the phys2bids original repo as upstream:

Reviewer: ```suggestion
	# we also recommend to set up the phys2bids original repo as upstream.
	# this way you can always sync your branches with the command git pull upstream master.
```

	git remote add upstream https://github.com/physiopy/phys2bids.git

> 	# this way you can always sync your branches with the command git pull upstream master 

Reviewer: ```suggestion
```

Reviewer: I'd either move this up or write the `git pull upstream master` command after the comment.

    If you want to check, type ``python --version`` in a terminal.

> Linux, Mac and Windows developer installation

Reviewer 2: ```suggestion
Linux, Mac, and Windows developer installation
```

> ---------------------------------------------

Reviewer 2: ```suggestion
----------------------------------------------
```

Move into the ``phys2bids`` folder and execute the command::

    pip3 install -e .[all]

This will install:

    - ``phys2bids`` as an editable package, which means that you can modify the program and run it without having to reinstall it every time!

>     - All the ``phys2bids`` dependencies.

Reviewer 2: ```suggestion
    - All ``phys2bids`` dependencies.
```

>     - All the **interface** modules, that deal with the interfaces for all the file formats.

Reviewer 2: ```suggestion
    - All **interface** modules, that deal with the interfaces for all the file formats.
```

>     - All the **style** modules, such as ``flake8``, to help you linter the code!

Reviewer 2: ```suggestion
    - All **style** modules, such as ``flake8``, to help you linter the code!
```

>     - All the **documentation** modules, like ``sphinx``, so that you can build the docs locally before submitting them.

Reviewer 2: ```suggestion
    - All **documentation** modules, like ``sphinx``, so that you can build the docs locally before submitting them.
```

>     - All the **test** modules, like ``pytest``, in order for you to test your code locally before committing it!

Reviewer 2: ```suggestion
    - All **test** modules, like ``pytest``, in order for you to test your code locally before committing it!
```

	- All the **style** modules, such as ``flake8``, to help you linter the code!
	- All the **documentation** modules, like ``sphinx``, so that you can build the docs locally before submitting them.
	- All the **test** modules, like ``pytest``, in order for you to test your code locally before committing it!

Check your installation!
^^^^^^^^^^^^^^^^^^^^^^^^

Type the commands::

	cd phys2bids/tests

>     pytest

Reviewer: These are not aligned.

Reviewer 2: I still don't see them aligned, did you commit and push the changes?
```suggestion
	cd phys2bids/tests
	pytest
```

> This will execute locally all test and check your phys2bids installation works properly

Reviewer: ```suggestion
This will execute the tests locally and check that your phys2bids installation works properly.
```

	test_integration.py ...                                                                                                                              [ 32%]
	test_phys2bids.py ...                                                                                                                                [ 37%]
	test_physio_obj.py .......                                                                                                                           [ 49%]
	test_txt.py ..................                                                                                                                       [ 78%]
	test_utils.py ...........                                                                                                                            [ 96%]
	test_viz.py .x                                                                                                                                       [100%]

	================================================================= short test summary info =================================================================
	XFAIL test_viz.py::test_plot_trigger

> 	======================================================== 60 passed, 1 xfailed in 142.58s (0:02:22) ========================================================

Reviewer 2: It's a great idea to run tests locally after installation, unluckily in this moment that means adding a bit more explanation about  the XFAIL status. Could you add a couple of lines to comment it, explaining why that happens and saying not to worry about it (but still worry about other errors)?

Author: Yep, I only added it in the contributors part because it needs all the dependencies

Author: The xfail error is commented now
